tests/routers/test_files.py

```
@@ -111,134 +110,152 @@ def test_list_files_no_filters(authenticated_client: TestClient, file_inventory_
-    assert len(files) == 1
-    assert "document" in files[0]["file_path"]
-
-
-def test_list_files_filter_by_extension(authenticated_client: TestClient, file_inventory_factory, tmp_path):
-    """Test filtering files by extension."""
-    file_inventory_factory(str(tmp_path / "file1.txt"), file_extension=".txt")
-    file_inventory_factory(str(tmp_path / "file2.jpg"), file_extension=".jpg")
-
-    response = authenticated_client.get("/api/v1/files?extension=.jpg")
+@pytest.mark.parametrize(
+    "setup_kwargs, filter_query, expected_path_part, expected_attr, expected_value",
```

Ruff (0.14.14)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)